### ISSUE TITLE
Missing the dropped hash rate from the logs

### ISSUE THREAD

**Issue by borzaka** [JerichoJones/JJ-s-XMR-STAK-HashRate-Monitor-and-Restart-Tool#5]
In the log:
`2017-12-02_11:38	Restarting after 35 Min - Hash rate dropped from 4411.6 H/s to  H/s`

Full log:
```
2017-12-02_11:01	Script Started
2017-12-02_11:01	XMR-STAK process was not found
2017-12-02_11:01	Running Driver Reset
2017-12-02_11:01	Video driver disabled
2017-12-02_11:01	Video driver enabled
2017-12-02_11:01	Video driver reset completed
2017-12-02_11:01	Starting C:\Users\Attila\Downloads\OverdriveNTool.exe -r0 -r1 -p0Vega64 -p1Vega64 
2017-12-02_11:01	Setting Env Variables for STAK
2017-12-02_11:01	Env Variables for STAK have been set
2017-12-02_11:01	Starting STAK...
2017-12-02_11:01	Waiting for STAK HTTP daemon to start
2017-12-02_11:01	STAK started successfully
2017-12-02_11:01	Waiting for hash rate to stabilize
2017-12-02_11:03	Hash rate stabilized
2017-12-02_11:03	Starting Hashrate: 4411.6 H/s	Drop Target Hashrate: 4111.6 H/s
2017-12-02_11:03	Hash monitoring has begun
2017-12-02_11:38	Restarting after 35 Min - Hash rate dropped from 4411.6 H/s to  H/s
2017-12-02_11:39	Script Ended
```

_Version: 2.7
Release Date: 2017-11-19_

**Comment by JerichoJones** [comment]
"Starting Hashrate: 4411.6 H/s	Drop Target Hashrate: 4111.6 H/s"
"Restarting after 35 Min - Hash rate dropped from 4411.6 H/s to  H/s"
I am adding more error code to v3.1 to see if it that catches this.

**Comment by JerichoJones** [comment]
Code has been written and added to v3.1. I have started testing the new code.

**Comment by borzaka** [comment]
I can confirm, that in the current master branch the dropped hash rate is no longer missing:

```
2017-12-03_23:02	Starting Hashrate: 2418.0 H/s	Drop Target Hashrate: 2218.0 H/s
2017-12-03_23:02	Hash monitoring has begun
2017-12-03_23:04	Restarting after 2 Min - Hash rate dropped from 2418.0 H/s to 2053.8 H/s
```

**Comment by JerichoJones** [comment]
Great! I'm going to close this then.

**Comment by borzaka** [comment]
This bug is still present in 3.0, and I think I found the cause; it's not a regular hash drop, it's probably an overclock/card failure.

```
2017-12-05_10:15 Starting Hashrate: 4599.2 H/s	Drop Target Hashrate: 4299.2 H/s
2017-12-05_10:15 Hash monitoring has begun
2017-12-05_11:19 Restarting after 1 Hours 4 Min - Hash rate dropped from 4599.2 H/s to  H/s
```

When the card overclock fails, the hashrate drops to zero; to `(na)` as seen in XMR-STAK, and `(na)` is not a number.

**Comment by JerichoJones** [comment]
It should be fixed in v3.1 or higher. I added several checks.
Are we connected?
Are we getting array data?
Is our hash total an integer
Is our hash total not zero
